src/main/java/uk/gov/hmcts/ccd/AccessManagementServiceConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.ccd;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import uk.gov.hmcts.reform.amlib.AccessManagementService;
+import uk.gov.hmcts.reform.amlib.DefaultRoleSetupImportService;
+
+import javax.sql.DataSource;
+
+@Configuration
```

Comment:
Do not we need this section here:

```
@EnableTransactionManagement
@EnableJpaRepositories(
    entityManagerFactoryRef = "amsEntityManagerFactory",
    transactionManagerRef = "amsTransactionManager",
    basePackages = { "uk.gov.hmcts.ccd.definition.store.accessmanagement", "uk.gov.hmcts.reform.amlib" }
)
```

See how it was done for https://github.com/hmcts/ccd-definition-store-api/pull/323 in `AccessManagementExportConfiguration`.

Reply:
We are creating the Hikari datasource and passing it on to the AM Library. We are not using any explicit transaction manager. Can you please let me know if a transaction manager would be mandatory in this case? I will improve the code accordingly